core/services/ocr2/plugins/directrequestocr/test/internal/testutils.go

```diff
@@ -99,6 +103,52 @@ func SetConfig(t *testing.T, owner *bind.TransactOpts, oracleContract *ocr2dr_or
 	require.NoError(t, err)
 }
 
+func SetRegistryConfig(t *testing.T, owner *bind.TransactOpts, registryContract *ocr2dr_registry.OCR2DRRegistry, oracleContract *ocr2dr_oracle.OCR2DROracle) {
+	var maxGasLimit =  uint32(1_000_000)
+	var stalenessSeconds =  uint32(86_400)
+	var gasAfterPaymentCalculation = big.NewInt(21_000 + 5_000 + 2_100 + 20_000 + 2 * 2_100 - 15_000 + 7_315)
+	var weiPerUnitLink =  big.NewInt(5000000000000000)
+	var gasOverhead =  uint32(500_000)
+
+	_, err := registryContract.SetConfig(
+		owner,
+		maxGasLimit,
+		stalenessSeconds,
+		gasAfterPaymentCalculation,
+		weiPerUnitLink,
+		gasOverhead,
+	)
+	require.NoError(t, err)
+
+	var senders []common.Address
+	senders[0] = oracleContract.Address()
+	_, err = registryContract.SetAuthorizedSenders(
+		owner,
+		senders,
+	)
+	require.NoError(t, err)
+}
+
+func CreateAndFundSubscriptions(t *testing.T, owner *bind.TransactOpts, numClients int64, linkToken *link_token_interface.LinkToken, registryContract *ocr2dr_registry.OCR2DRRegistry, clientContracts []deployedClientContract) (subscriptionId uint64) {
+	_, err := registryContract.CreateSubscription(owner)
+	require.NoError(t, err)
+
+	subscriptionID := uint64(1)
+
+	for i := 0; i < len(clientContracts); i++ {
+		_, err = registryContract.AddConsumer(owner, subscriptionID, clientContracts[i].Address)
+		require.NoError(t, err)
+	}
+
+	data, err := utils.ABIEncode(`[{"type":"uint64"}]`, subscriptionID)
+	require.NoError(t, err)
+	
+	amount := big.NewInt(0).Mul(big.NewInt(numClients), big.NewInt(1e18)) // 1 LINK per client
```

Comment:
You have len(clientContracts) so you don't need to pass numClients separately.

Reply:
Good point